pages/success-banner/index.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import Head from 'next/head';
+
+/**
+ * @summary Make and return an event method for a given event description string
+ * @function
+ * @private
+ *
+ * @param {String} eventDecs - event description
```

Comment:
Just noticed that this says `eventDecs` when it should say `eventDesc` - could you squeeze it into #77 @Shou :)